src/features/ranking/index.tsx

```
@@ -22,7 +23,22 @@ const RankingPage = ({ ranking }: RankingPageProps) => {
         {ranking.map((player, index) => (
-          <PlayerCard player={player} chara={player.chara} ranking={player.rank} key={index} />
+          <PlayerCard player={player} chara={player.chara} key={index}>
+            <div className="flex">
+              <div className="text-sm text-gray-600">{player.rank}位</div>
+              <div className="text-sm text-gray-600 ml-1 truncate">
+                {`| ${player.points}P`}
+                <span className="min-w-0 mx-2 border border-1 border-gray p-[1] px-2 m-1 rounded text-[10px] truncate">
+                  <span
+                    className="truncate"
+                    dangerouslySetInnerHTML={{
+                      __html: player.achivement.markup || player.achivement.title,
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)